app/js/deterministicResults/valueProfileTableDirective.html

```diff
@@ -1,4 +1,4 @@
-<table id="{{ case }}-value-profile-table" class="rank-table">
```

Comment:
is the class used anywhere else? otherwise might as well delete it from the scss as well

Reply:
Yes, inside the rankAccessibilitiesDirective